Codeinwp/themeisle-companion
# Update old php-http/discovery composer dependency

The error came from Codeinwp/themeisle-content-forms repo. We used an old version of mailerlite/mailerlite-api-v2-php-sdk package which was using an old version of php-http/discovery. I manage to update them and the issue should be fixed now.

###Testing instructions:
- Install this version of OrbitFox 
- Install Elementor
- Create a new page and add those widgets to your page https://vertis.d.pr/diVjBa
- Check that everything works on PHP 7.4 and 8
- Install this plugin https://wordpress.org/plugins/wp-mail-smtp/ and check the slack messages, I've sent you some login credentials
- After you set up that plugin, try sending a message via the Contact form widget from elementor. You should receive a mail in your inbox.

Closes #747
